### ISSUE TITLE
Problems by using plugin.tx_sfeventmgt.view

### ISSUE THREAD

**Issue by Cretection** [derhansen/sf_event_mgt#259]
Hi,
thanks for your Extension. I tried this: 
```
plugin.tx_sfeventmgt {
  view {
    templateRootPath = EXT:myext/Resources/Private/Plugins/sf_event_mgt/Templates/
    partialRootPath = EXT:myext/Resources/Private/Plugins/sf_event_mgt/Partials/
    layoutRootPath = EXT:myext/Resources/Private/Plugins/sf_event_mgt/Layouts/
  }
}
```
But it changes nothing! And I tried every possible configuration.

I tried the other way you described in the manuel:
```
plugin.tx_sfeventmgt {
  view {
    templateRootPaths {
      0 = {$plugin.tx_sfeventmgt.view.templateRootPath}
      1 = EXT:myext/Resources/Private/Plugins/sf_event_mgt/Templates/
    }
    partialRootPaths {
      0 = {$plugin.tx_sfeventmgt.view.partialRootPath}
      1 = EXT:myext/Resources/Private/Plugins/sf_event_mgt/Partials/
    }
    layoutRootPaths {
      0 = {$plugin.tx_sfeventmgt.view.layoutRootPath}
      1 = EXT:myext/Resources/Private/Plugins/sf_event_mgt/Layouts/
    }
  }
}
```

this is working for me. But whats up with the first variant?

**Comment by derhansen** [comment]
I've tried to reproduce your problem with variant 1 with no success. I was able change to path to the templates with the code snippet you provided. I just changed the pathes to a location with the template/layout/partial files and it works fine.

**Comment by derhansen** [comment]
No feedback for over 30 days, closing issue.

**Comment by derhansen** [comment]
Does only apply to TYPO3 7.6

**Comment by Cretection** [comment]
Sorry, that I am not Dome Feedback. I had no Time for more testings and I am still working with my second variant

**Comment by derhansen** [comment]
Issue is resolved with #289
